yb-voyager/src/query/queryparser/helpers_protomsg.go

```diff
@@ -420,28 +412,44 @@ func ProtoAsSelectStmt(msg protoreflect.Message) (*pg_query.SelectStmt, error) {
 	return selectStmtNode, nil
 }
 
+func ProtoAsCTENode(msg protoreflect.Message) (*pg_query.CommonTableExpr, error) {
+	protoMsg, ok := msg.Interface().(proto.Message)
```

Comment:
I found out recently you don't need to do two steps. 
You can convert it in one step directly. 

`msg.Interface().(*pg_query.xxx)`